lib/trino-filesystem/src/test/java/io/trino/filesystem/AbstractTestTrinoFileSystem.java

```diff
@@ -510,7 +510,11 @@ void testOutputFile()
 
             if (isCreateExclusive()) {
                 // re-create without overwrite is an error
-                assertThatThrownBy(outputFile::create)
+                assertThatThrownBy(() -> {
```

Comment:
Let's change this to
```java
assertThatThrownBy(() -> outputFile.create().close())

Reply:
`assertThatThrownBy(outputFile::create)`?

Reply:
No, we need to ensure that it's closed. There is no guarantee that we throw on create.